content/widgets.md

```diff
@@ -58,6 +58,18 @@
 * Update the preferred content size
 * Wait for `viewWillTransition(to:with:)` to be called and update UI accordingly
 
+## Detecting if Device is Locked
+
+1. The first time your main app starts create a dummy file (let's call it ```ProtectionMonitor.dummy```) in a shared group container which is accessible for both the main app and the widget.
+	* You can create this file with ```NSFileManager.createFile(atPath:, contents:, attributes:)```
+2. Make sure the file is protected by iOS Data Protection.
+	* You can ensure by passing the dictionary ```{NSFileProtectionKey: NSFileProtectionComplete}``` as the attributes parameter in the above method
+	* This step requires that Data Protection is enabled for your main app
```

Comment:
This should probably be the vey first sentence, above 1.: Lock detection requires your app to enable Data Protection in its entitlements.